docs/csharp/programming-guide/concepts/covariance-contravariance/variance-in-generic-interfaces.md

```diff
@@ -22,6 +24,12 @@ ms.assetid: 4828a8f9-48c0-4128-9749-7fcd6bf19a06
 
 - <xref:System.IComparable%601> (T is contravariant)
 
+The following variant interfaces were added in .NET Framework 4.5:
+
+- <xref:System.Collections.Generic.IReadOnlyList%601> (T is contravariant)
+
+- <xref:System.Collections.Generic.IReadOnlyCollection%601> (T is contravariant)
+
 Covariance permits a method to have a more derived return type than that defined by the generic type parameter of the interface. To illustrate the covariance feature, consider these generic interfaces: `IEnumerable<Object>` and `IEnumerable<String>`. The `IEnumerable<String>` interface does not inherit the `IEnumerable<Object>` interface. However, the `String` type does inherit the `Object` type, and in some cases you may want to assign objects of these interfaces to each other. This is shown in the following code example.
 
 ```csharp
```

Comment:
unrelated to your PR: on line 40, it's VB and not C# that has an `Option Strict On` setting. Could you change this to read "...a compilation error in C# and, if `Option Strict` is on, in Visual Basic."

Reply:
ah, good catch, will do.